### ISSUE TITLE
[4.3.3] Tabs are always light themed

### ISSUE THREAD

**Issue by tom2199** [crash0verride11/DarkLight-qBittorent-WebUI#3]
I semi-sucessfully installed this theme, but I have aproblem with the tabs, here is a screenshot:
![image](https://user-images.githubusercontent.com/9537697/105508712-fd702100-5ccc-11eb-91fb-128f3fa1763f.png)
I am using qBittorrent version 4.3.3 on arm64, if that's important. I've uploaded my theme folder, maybe I did something wrong? (Though I tried to reproduce every step of the installation tutorial)
[theme.zip](https://github.com/crash0verride11/DarkLight-qBittorent-WebUI/files/5856740/theme.zip)

**Comment by crash0verride11** [comment]
I see another issue opened this week so there may be an issue with the dynamictable.js generates the tables or newer changes in the source www folder. The other bug was also related to edge though so likely a browser specific issue.

Are you using the newer chromium based edge? and have tried any of the other chrome browsers? (google chrome, brave etc)

**Comment by tom2199** [comment]
Yes I use the chromium based Edge and no, I haven't tested other browsers besides Edge and Firefox. Weirdly enough, I later had the same issue with Firefox, but it went away after a restart, so I don't know what's going on here. Thanks for looking into this!

**Comment by AyoKeito** [comment]
I've had the same problem in Firefox 96 and it was solved via F12 menu:
Press F12, go to network menu, check "disable cache", refresh page, uncheck "disable cache". That was enough to make it work properly, while ctrl+f5 and shift+refresh were not doing anything to solve the problem.
